src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -896,6 +895,15 @@ def validate_params(self):
                 manifest.is_folder_allowed()
             ), f"Invalid value for `folder` {manifest.folder} must be one of {manifest.get_allowed_folders()}"
 
+    def create_single_custom_manifest(self, filename: str = None, folder: str = None, base64_content: str = None):
+        return self.api_client.create_custom_manifest(self.id, filename, folder, base64_content)
```

Comment:
The call should be
```python
        return self.api_client.create_custom_manifest(self.id, folder, filename, base64_content)
```
need to swap `filename` and `folder` position

Reply:
done